triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -105,6 +133,67 @@ class Server(triggerDao: RunningTriggerDao) {
     triggerDao.persistPackages(encodedDar)
   }
 
+  private def triggerRunnerName(triggerInstance: UUID): String =
+    triggerInstance.toString ++ "-monitor"
+
+  private def getRunner(triggerInstance: UUID): Option[ActorRef[TriggerRunner.Message]] =
+    ctx
+      .child(triggerRunnerName(triggerInstance))
+      .asInstanceOf[Option[ActorRef[TriggerRunner.Message]]]
+
+  private def startTrigger(
```

Comment:
Did any of the methods here change or did you just shuffle them around? Unfortunately hard to spot in the diff.

Reply:
Just shuffled!